_posts/2014-03-21-29-minutes.html

```diff
@@ -15,11 +15,11 @@
 ---
 
 <p>I want to relate a great little thing that happened here at <a href="http://gsa.gov/">18th and F</a> yesterday.</p>
-<p>At 11am yesterday morning, members of the GSA.gov team, Sarah Hyder and Patrick Son, met with Hillary Hartley, Alison Rowland, and me to talk about how to integrate <a href="https://fbopen.gsa.gov">FBOpen</a> into GSA&#8217;s mobile website, <a href="http://m.gsa.gov">m.gsa.gov</a>, on its &#8220;<a href="http://m.gsa.gov/m/#!/buy-sell/how-to-sell">How To Sell to the Government</a>" page.</p>
+<p>At 11am yesterday morning, members of the GSA.gov team, Sarah Hyder and Patrick Son, met with Hillary Hartley, Alison Rowland, and me to talk about how to integrate <a href="https://fbopen.gsa.gov">FBOpen</a> into GSAs mobile website, <a href="http://m.gsa.gov">m.gsa.gov</a>, on its <a href="http://m.gsa.gov/m/#!/buy-sell/how-to-sell">How To Sell to the Government</a>" page.</p>
```

Comment:
Ah! I did *not* know that was the encoding for an apostrophe. I kept pulling down from the site and couldn't figure out why it was replacing all my punctuation with a jumble of symbols! Newbie mistake, haha.